src/app/app.component.ts

```diff
@@ -93,11 +93,13 @@ export class AppComponent implements OnInit {
     this.store.dispatch(new sharedActions.FetchProjects());
     this.store.dispatch(new sharedActions.FetchJobs());
     this.store.dispatch(new sharedActions.FetchDesigns());
+    this.store.dispatch(new sharedActions.FetchExperiments());
 
     combineLatest(this.store.pipe(select((store) => store.interactiveMap.selectedSpecies)),
       this.store.pipe(select((store) => store.shared.modelHeaders))).subscribe(([species, models]) => {
       if (species && models.length > 0) {
         this.store.dispatch(new SelectFirstModel(species, models));
+        this.store.dispatch(new SetModelDataDriven(models.find((model) => model.id === 10)));
```

Comment:
- Don't use the database id, use the name of the model. Database ids are likely to change for on premise installations and in dev envs
- Only prefer iJO1366 for e. coli, not all species
- Handle the case if the model doesn't exist, otherwise this would crash

Reply:
Well, when we create a card is always created with the e.coli, in case that .find dont find any models, the result will be have undefined in the model of data driven cards... How can we handle this case?

Reply:
I don't think we should assume any species to be available on all installations. For other species if there isn't a preferred we can probably just select the first model.